### ISSUE TITLE
Need to add classes to input divs

### ISSUE THREAD

**Issue by aldeed** [meteoric/autoform-ionic#3]
I need to add additional classes, namely `item-text-wrap`, to the `div` generated by "boolean-checkbox" type, and potentially by other checkbox and radio types. Like this:

``` html
<div class="item item-checkbox item-text-wrap">
    <label class="checkbox">
      <input type="checkbox" value="" name="allStatementsTrue" data-schema-key="allStatementsTrue">
    </label>
    Check here if ALL of the above statements are TRUE.
</div>
```

I'm thinking maybe just add `div-class="item-text-wrap"` attribute and in the theme templates, append that to the "item item-checkbox" classes.

If you think that's a good way to do it, I can do a PR.

**Comment by nickw** [comment]
Actually I think we can safely add that class to all form fields. I've noticed myself needing it quite often.

**Comment by aldeed** [comment]
OK, I wasn't sure if there would be cases where people want the ellipses instead of wrapping, but truthfully I can't think of any time I would want that. I'll submit a PR to add it as default.

**Comment by aldeed** [comment]
PR submitted

**Comment by nickw** [comment]
Thanks, published 1.0.5
